airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/error_handlers/error_handler.py

```diff
@@ -26,7 +26,7 @@ def max_retries(self) -> Union[int, None]:
         pass
 
     @abstractmethod
-    def should_retry(self, response: requests.Response) -> ResponseStatus:
+    def interpret_response(self, response: requests.Response) -> ResponseStatus:
```

Comment:
I renamed this method because in `simple_retriever.parse_response()` we're not just getting the retryability, we're interpreting what the response status which is probably a more accurate name